images/base/files/etc/containerd/config.toml

```diff
@@ -27,7 +27,7 @@ version = 2
 
 [plugins."io.containerd.grpc.v1.cri"]
   # use fixed sandbox image
-  sandbox_image = "k8s.gcr.io/pause:3.6"
+  sandbox_image = "registry.k8s.io/pause:3.6"
```

Comment:
@BenTheElder we should be able to patch this for CI from the config, right?

```
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
  - role: control-plane
containerdConfigPatches:
  - |-
    [plugins."io.containerd.grpc.v1.cri".]
      sandbox_image = "registry.k8s.io/pause:3.6"
```

Reply:
https://github.com/kubernetes-sigs/kind/blob/4cd81b46b9b6e3015ca64331a6e24e0c301c7e7b/hack/ci/e2e-k8s.sh#L138-L152

Reply:
> @BenTheElder we should be able to patch this for CI from the config, right?

yes, but that won't apply to image builds.